plasmapy/dispersion/numerical/hollweg_.py

```diff
@@ -262,7 +262,7 @@ def hollweg(
     if not (theta.ndim == 0 or theta.ndim == 1):
```

Comment:
```suggestion
    if theta.ndim not in (0, 1):
```

This suggestion is a little out of scope, and nothing is wrong with the original code, but this is the more Pythonic way.